ChangeLog

```diff
@@ -1,3 +1,7 @@
+2018-06-09 Stjepan Brkić <stjepanbrkicc@gmail.com>
+	* Source/CALayer.m:
+	Fix failing build by adding CGFloat _contentsScale ivar.
```

Comment:
This totally belongs in a separate PR.

Reply:
This was already merged

Reply:
That's fine then, but for future reference, it still does belong in a separate PR.